safe/impact_functions/core.py

```diff
@@ -674,8 +658,9 @@ def get_admissible_plugins(keywords=None):  # , name=None):
 
 
 def parse_single_requirement(requirement):
-    '''Parse single requirement from impact function's doc to category,
-        subcategory, layertype, datatype, unit, and disabled.'''
+    """Parse single requirement from impact function's doc to category,
```

Comment:
It's better to have a one-line explanation of the function and explain more after that as a different paragraph